src/models/controller.ts

```diff
@@ -0,0 +1,94 @@
+import express from 'express'
+const router = express.Router()
+import { v4 as uuid } from 'uuid'
+import { Request, Response, NextFunction } from 'express'
+import UserService from '../models/user'
+
+interface RequestBody {
+  name: string
+  age: number
+  id: string
+  path: string
+  nameOfFile: string
+}
+
+let user: UserService = new UserService()
+
+export const initializeUser = async (
+  req: Request<{}, {}, RequestBody, {}>,
+  res: Response
+) => {
+  try {
+    await user.init()
+    const { nameOfFile } = req.body
+    if (nameOfFile) {
+      res.status(200).send({ message: 'Dir/file is already exists' })
+    }
+    res.status(200).send({ message: 'Dir/file has been successfuly created' })
+  } catch (error) {
+    res.status(422).send({ error })
+  }
+}
+export const createUser = async (
+  req: Request<{}, {}, RequestBody, {}>,
+  res: Response
+) => {
+  try {
+    const { name, age } = req.body
+    const id = uuid()
+    await user.create({ name, age, id })
+    res.status(200).send({ message: `User was successfuly created ` })
+  } catch (error) {
+    res.status(422).send({ error })
+  }
+}
+
+export const getAllUsers = async (
+  req: Request<{}, {}, RequestBody, {}>,
```

Comment:
In this case you don't need RequestBody, because you will not send body with these fields. Also you can implement pagination for this controller (see nodejs course) and specifiy interface RequestQuery { limit, skip, sort }

Reply:
Don't know how to do in typeScript